src/components/SelectionList/BaseSelectionList.tsx

```diff
@@ -278,7 +281,7 @@ function BaseSelectionList<TItem extends ListItem>(
         const indexOffset = section.indexOffset ? section.indexOffset : 0;
         const normalizedIndex = index + indexOffset;
         const isDisabled = !!section.isDisabled || item.isDisabled;
-        const isItemFocused = !isDisabled && focusedIndex === normalizedIndex;
+        const isItemFocused = !isDisabled && (focusedIndex === normalizedIndex || itemToHighlight?.has(item.keyForList));
```

Comment:
TS is failing here, I think we need a default in the case `itemToHighlight` is `null`